pydis_site/apps/api/tests/test_users.py

```diff
@@ -199,13 +200,15 @@ def test_multiple_users_patch(self):
         url = reverse("api:bot:user-bulk-patch")
         data = [
             {
+                "alts": [],
```

Comment:
Does this imply that setting this field is required when creating/updating? We probably don't want this, as it would mean we need to update all the places in bot that does this to also set it to `[]`.

We probably want it default to `[]`, and have a test to make sure it works if this field isn't passed in the body

Reply:
This isn't the case, I read wrong.